cmd/autoscale-scheduler/main.go

```diff
@@ -53,7 +60,12 @@ func runProgram() (err error) {
 		return err
 	}
 
-	command := app.NewSchedulerCommand(app.WithPlugin(plugin.Name, plugin.NewAutoscaleEnforcerPlugin(ctx, conf)))
+	// The normal scheduler outputs to klog, and there isn't *really* a way to stop that. So to make
+	// everything fit nicely, we'll redirect it to zap as well.
+	redirectKlog(logger.Named("klog"))
```

Comment:
Noticed that the scheduler's klog output still isn't being redirected... need to fix that before merging. Maybe I need to supply the klog instance

Reply:
Thought it might've been that for a minute. Did some digging, looks like klog _by default_ has `LogToStderr` set to `true`, which completely bypasses the output redirection :confused:

But calling `klog.LogToStderr(false)` fixes it, so... should be ok I think.